mlflow/protos/service.proto

```diff
@@ -822,6 +817,13 @@ message SearchRuns {
   // Maximum number of runs desired. Max threshold is 50000
   optional int32 max_results = 5 [default = 1000];
 
+  // List of columns to be ordered by, including attributes, params, metrics, and tags with an
+  // optional "DESC" or "ASC" annotation, where "ASC" is the default.
+  // Example: ["params.input DESC", "metrics.alpha ASC", "metrics.rmse"]
+  // Tiebreaks are done by start_time DESC followed by run_id for runs with the same start time
+  // (and this is the only ordering criterion if order_by is not provided).
```

Comment:
excruciating small *nit*

and this is the only ordering criterion ... -> and this is the default ordering criterion ...